### PR TITLE
upgrade java chart version

### PR DESCRIPTION
**Pull request by klawani**
### Change description ###

use chart-java version 2.13.1  to use cpuRequests: '250m' by default
remove redundant config values of  PAY_URL & FEES_URL

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: charts/cmc-claim-store/values.aat.template.yaml [hunk 1]
```diff
@@ -5,8 +5,6 @@ java:
     PDF_SERVICE_URL: http://cmc-pdf-service-java.software-engineering
     SEND_LETTER_URL: http://rpe-send-letter-service-java.software-engineering
     DOCUMENT_MANAGEMENT_URL: http://dm-store-java.ccd
-    PAY_URL: false
-    FEES_URL: false
```
Comment (andrewwa-kainos) [inline charts/cmc-claim-store/values.aat.template.yaml]: Please don't remove these.

File: charts/cmc-claim-store/values.preview.template.yaml [hunk 2]
```diff
@@ -35,8 +35,6 @@ java:
     IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
     PDF_SERVICE_URL: http://cmc-pdf-service-aat.service.core-compute-aat.internal
     DOCUMENT_MANAGEMENT_URL: http://dm-store-aat.service.core-compute-aat.internal
-    PAY_URL: false
-    FEES_URL: false
```
Comment (andrewwa-kainos) [inline charts/cmc-claim-store/values.preview.template.yaml]: Please don't remove these

File: charts/cmc-claim-store/values.aat.template.yaml (hunk 1 above)
Comment (klawani) [reply charts/cmc-claim-store/values.aat.template.yaml]: don't worry about it ... it's already in values.yaml file which is the default values

File: charts/cmc-claim-store/values.preview.template.yaml (hunk 2 above)
Comment (klawani) [reply charts/cmc-claim-store/values.preview.template.yaml]: same as in aat. It's default in values.yaml file already ...

Comment (sabahirfan) [reply charts/cmc-claim-store/values.preview.template.yaml]: These are the default values in `values.yaml`, we should have only those here which we want to override defaults.

**Pull request merged**